resources/views/profile.blade.php

```diff
@@ -14,12 +14,12 @@
     <div style="background-size:cover; background: url({{ Voyager::image( Voyager::setting('admin_bg_image'), config('voyager.assets_path') . '/images/bg.jpg') }}) center center;position:absolute; top:0; left:0; width:100%; height:300px;"></div>
     <div style="height:160px; display:block; width:100%"></div>
     <div style="position:relative; z-index:9; text-align:center;">
-        <img src="{{ Voyager::image( Auth::user()->avatar ) }}" class="avatar"
+        <img src="@if( strpos(auth()->user()->avatar, 'http://') === false && strpos(auth()->user()->avatar, 'https://') === false){{ Voyager::image( auth()->user()->avatar ) }}@else{{ auth()->user()->avatar }}@endif" class="avatar"
              style="border-radius:50%; width:150px; height:150px; border:5px solid #fff;"
-             alt="{{ Auth::user()->name }} avatar">
-        <h4>{{ ucwords(Auth::user()->name) }}</h4>
-        <div class="user-email text-muted">{{ ucwords(Auth::user()->email) }}</div>
-        <p>{{ Auth::user()->bio }}</p>
-        <a href="{{ route('voyager.users.edit', Auth::user()->id) }}" class="btn btn-primary">Edit My Profile</a>
+             alt="{{ auth()->user()->name }} avatar">
+        <h4>{{ ucwords(auth()->user()->name) }}</h4>
+        <div class="user-email text-muted">{{ ucwords(auth()->user()->email) }}</div>
+        <p>{{ auth()->user()->bio }}</p>
```

Comment:
no need for `isset` as the column will be avail in the db, so if no value it will empty